### ISSUE TITLE
Staff perms

### ISSUE THREAD

**Issue by OEillet-Rissa** [mixxit/solinia3-core#339]
/ban /tempban /mute
We had an issue with a player biting without permission, Rissa approched (yes im in third person) and told him to stop, he instantly bit her and then she attempted to ban, got told improper parameters, them when adjusted got flat out denied and Dernable had to handle the ban. Please and sorry!!!

**Comment by Silternius** [comment]
Yes, that's because those are moderator commands, should be only moderators and admins that have access if I'm not mistaken.
